src/assets/Vars.js

```diff
@@ -41,23 +41,23 @@ export const SQUAD_VELOCITY = 109.890938;
 export const SQUAD_MAX_DISTANCE = 1232;
 
 export const PS_8CM_NAME = "GER 8cm";
-export const PS_8CM_VELOCITY = 116.179939;
-export const PS_8CM_MAX_DISTANCE = 1377;
+export const PS_8CM_VELOCITY = 119.93;
+export const PS_8CM_MAX_DISTANCE = 1467;
 
 export const PS_3INCH_NAME = "BRIT 3″";
-export const PS_3INCH_VELOCITY = 116.179939;
-export const PS_3INCH_MAX_DISTANCE = 1377;
+export const PS_3INCH_VELOCITY = 119.93;
+export const PS_3INCH_MAX_DISTANCE = 1467;
 
 export const PS_4INCH_NAME = "BRIT 4″";
-export const PS_4INCH_VELOCITY = 160.598197;
-export const PS_4INCH_MAX_DISTANCE = 2632;
+export const PS_4INCH_VELOCITY = 166.94;
+export const PS_4INCH_MAX_DISTANCE = 2818;
 
 export const PS_6CM_NAME = "US 6cm";
-export const PS_6CM_VELOCITY = 116.179939;
-export const PS_6CM_MAX_DISTANCE = 1377;
+export const PS_6CM_VELOCITY = 119.93;
+export const PS_6CM_MAX_DISTANCE = 1467;
 
 export const GRAVITY = 9.8;
-export const MIL_TO_DEG_FACTOR = 360 / 6400;
+export const MIL_TO_DEG_FACTOR = 360.0 / 6283.1853;
```

Comment:
Yeah I'll release it as soon as this is cleared up.

Do you know that they are using these values ingame?

I don't know which one they use internally, but so far I relied on them using NATO mils, see here: https://en.wikipedia.org/wiki/Milliradian#Definitions_for_maps_and_artillery

> For maps and artillery, three rounded definitions are used which are close to the real definition, but more easily can be divided into parts. The different map and artillery definitions are sometimes referred to as "angular mils", and are:
> 
> * 1/6400 of a circle in NATO countries.
> * ...

Reply:
I'll check with the default value in the afternoon and check if the results are suitable.

I'm not sure about NATO values in Post Scriptum, since NATO was founded in 1949.

Reply:
How will this impact **Squad** MC accuracy?

Reply:
I just check on both game.
It seems that Squad use NATO mil and Post Scriptum default milliradians ...

I'm asking PS devs about that

Reply:
@Endebert is it possible to use 1/6400 for Squad and 1/6283.1853 for Post Scriptum ?

Reply:
Yes you can, but you would need to rely on the flag that determines if you're in Squad or PS mode:

App.vue:
`// variable that determines PostScriptum mode`
`const postScriptum = false;`

So based on that bool flag you can determine which mil standard to use. Either you have to start passing this boolean around relevant methods. From the first glance that looks like a terrible idea as you would have to pass it through a lot of methods. 

Alternatively, you can maybe move the flag into Utils file all together. Can't exactly tell you if that will have any drastic side effects and whether you will break the app, but feel free to try. By moving the flag into Utils, you can simplify logic like this one in Map.vue: 
`      /** @type {MortarType[]} */`
  `mortarTypes: this.postScriptum ? Utils.getPSMortarTypes() : Utils.getSquadMortarTypes(),`